plugins/org.jboss.tools.mylyn.reddeer/src/org/jboss/tools/mylyn/reddeer/TestSupport.java

```diff
@@ -12,11 +16,17 @@
 import org.jboss.reddeer.swt.impl.shell.DefaultShell;
 import org.jboss.reddeer.swt.impl.tree.DefaultTree;
 import org.jboss.reddeer.swt.impl.tree.DefaultTreeItem;
+import org.jboss.reddeer.workbench.ui.dialogs.WorkbenchPreferenceDialog;
 
 public class TestSupport {
 	
+	/* Cannot reference "this" in a static context - the following cannot be used here:
+	 * Logger log = Logger.getLogger(this.getClass());
+	 */
+	private static final Logger log = Logger.getLogger(TestSupport.class);
+	
 	/* Test Setup part 1 */
-	public static List<TreeItem> mylynTestSetup1 (Logger log) {		
+	public static List<TreeItem> mylynTestSetup1 () {		
 		
```

Comment:
Please do not use prefix like this "*** step" use log.debug() or log.trace() and message without prefix instead